### ISSUE TITLE
Request: Read Fixed Addresses from DHCP config file

### ISSUE THREAD

**Issue by Julio-UAM** [Akkadius/glass-isc-dhcp#55]
It would be great if you could read all your fixed addresses from your config file and display them as the dynamic leases, so we can see at a glance if an scope is filling up.

**Comment by maglite4cell** [comment]
I agree this would be a huge benefit.  Most of what we do in our production env is reserved addresses and this would be a huge help to know what devices are assigned

**Comment by PartTimeJS** [comment]
> 
> 
> I agree this would be a huge benefit. Most of what we do in our production env is reserved addresses and this would be a huge help to know what devices are assigned

Same here. I have been looking at npm packages to see if there is an easy way to do this and maybe PR it. Worst case, the config needs a lot of parsing to list defined hosts. Maybe after the holidays I will have more time to dig in deeper if the repo dev doesn't get to it. Seeing as this request is over a year old, not looking too good.
